debian/control

```diff
@@ -0,0 +1,17 @@
+Source: ooniprobe-cli
+Maintainer: Federico Ceratto <federico@openobservatory.org>
+Section: devel
+Priority: optional
+Build-Depends: dpkg-dev, debhelper-compat (= 12)
+Standards-Version: 4.5.0
+Vcs-Browser: https://salsa.debian.org/go-team/packages/probe-cli
+Vcs-Git: https://salsa.debian.org/go-team/packages/probe-cli.git
```

Comment:
These two URLs strike me as unexpected

Reply:
Yes I am also not sure why these are here. Maybe it's some debian internal thing.